core/src/main/scala/io/qbeast/core/model/QuerySpace.scala

```diff
@@ -45,17 +45,10 @@ case class AllSpace(dimensionCount: Int) extends QuerySpace {
  * Describe the query range in the area included in [originalFrom,originalTo)
  * (inclusive, exclusive).
  *
- * @param originalFrom inclusive starting range
- * @param originalTo   exclusive ending query range
- * @param revision revision applied on this space
+ * @param from inclusive starting range
+ * @param to   exclusive ending query range
  */
-case class QuerySpaceFromTo(originalFrom: Point, originalTo: Point, revision: Revision)
-    extends QuerySpace {
-
-  require(originalFrom <= originalTo, "from point must be < then to point")
-  require(originalFrom.dimensionCount == originalTo.dimensionCount)
-  val from: Point = Point(revision.transform(originalFrom.coordinates))
-  val to: Point = Point(revision.transform(originalTo.coordinates))
+case class QuerySpaceFromTo(from: Point, to: Point) extends QuerySpace {
```

Comment:
we shouldn't expose from and to point. QuerySpace should not expose any detail about how we implement the intersectsWith methods. 

Second point, using the Point class is sub-optimal, as we might have 1 or many dimensions unbounded and using from is inconvinient. We should think about a better private structure to use, something that use either optional filtering or something similar

Reply:
Made some changes to the creation of the QuerySpace on last commits ae8b2c829e180b48870f0fc62ef32d3cf5769dfe 2acdae12b22ada61689ddc98bad4b773f0f08801